DataFormats/MuonReco/interface/EmulatedME0Segment.h

```diff
@@ -0,0 +1,77 @@
+#ifndef ME0RecHit_EmulatedME0Segment_h
```

Comment:
I think this should be defined in the same place as the ME0Segment.h (GemRecHit/)